src/js/shield_defs.js

```diff
@@ -2716,6 +2704,26 @@ export function loadShields(shieldImages) {
     Color.shields.blue
   );
 
+  // Myanmar
+  shields["MY:E"] = shields["my:federal"] = hexagonVerticalShieldYellow;
+
+  // Nepal
+  shields["np:national"] = roundedRectShield(
+    Color.shields.green,
+    Color.shields.white
+  );
+  shields["np:regional"] = roundedRectShield(
+    Color.shields.white,
+    Color.shields.black
+  );
+
+  // Philippines
+  shields["PH:N"] = homeDownShield;
+  shields["PH:E"] = homeDownShieldYellow;
+
+  // Turkey
+  shields["TR:motorway"] = hexagonVerticalShieldOrange;
```

Comment:
Nit: “Taiwan” sorts before “Turkey” (or “Türkiye”).

Reply:
That's intentional; it's sorted by ISO country code.